openshift/oadp-operator
# doc: How to create new release branch CI files

## Why the changes were made

The goal of this PR is to document the steps needed to create a new release branches, so no steps are missed next time we do it. In the future, this should be automated.

Fixes https://github.com/openshift/oadp-operator/issues/1227

## How to test the changes made

Read the [documentation](https://github.com/mateusoliveira43/oadp-operator/blob/doc/ci-new-branch/docs/developer/PROW_CI.md#creating-new-release-branches) and make sure it is clear. Check referenced PRs in linked issue to see if no steps were missed.
